lib/gli/commands/help_modules/command_help_format.rb

```diff
@@ -13,38 +13,38 @@ def initialize(command,app,sorter,synopsis_formatter_class,wrapper_class=TextWra
         end
 
         def format
-          command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
-          wrapper              = @wrapper_class.new(Terminal.instance.size[0],4)
+          @command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
```

Comment:
Not all Ruby warnings apply to 100% of the cases (exactly why it is a warning, not a failure). In this particular case, we get this sort of warning:
```
gli/lib/gli/commands/help_modules/command_help_format.rb:16: warning: assigned but unused variable - command_wrapper
```
This is clearly not a bug, because the local variable at the binding is used by the ERB template. But the VM has no way of knowing that until run time. Using instance variables in such cases, is a good option, because it avoids the warning.

But still, a clean room technique would be better. Just an example:
```ruby
require 'erb'

class CleanRoom
  def initialize attrs
    self.singleton_class.class_eval do
      attrs.keys.each { |key| attr_accessor key }
    end
    attrs.keys.each do |key|
      self.send(:"#{key}=", attrs[key])
    end
  end
  def render_erb template
    ERB.new(template).result(binding)
  end
end

clean_room = CleanRoom.new(
  var_a: 'a_value',
  var_b: 'b_value'
)

template = '<%= "#{var_a} / #{var_b}" %>'

puts clean_room.render_erb(template)

# >> a_value / b_value
```
The binding for ERB is now a dedicated object, with proper accessors. A clean room makes it explicit what should be accessible by the template. In the example, I used some metaprogramming to define singleton methods for accessors, so it does not get in the way, and can be used anywhere that a template is needed.